<launch>
    <node pkg="sesto_server_interface" name="sesto_server_interface_node" type="sesto_server_interface_node" output="screen">
        <remap from="tcp_socket/write" to="tcp_client/write"/>
        <remap from="tcp_socket/read" to="tcp_client/read"/>
        <remap from="multicast_socket/write" to="udp_multicaster/write"/>
        <remap from="multicast_socket/read" to="udp_multicaster/read"/>
    </node>

    <node name="udp_multicaster" pkg="socket_comms" type="udpMultiLine" output="screen" respawn="true">
        <param name="port" type="int" value="1024" />
        <param name="address" type="str" value="239.239.239.239" />
    </node>
    
    <node name="tcp_client" pkg="socket_comms" type="tcpClient" output="screen">
        <param name="port" value="1024" />
        <param name="freq" value="20"/>
		<param name="address" type="str" value="192.168.1.222" />
    </node>
</launch>
